packages/comms/src/services/wsdl/WsDfu/v1.63/WsDfu.ts

```diff
@@ -1022,442 +942,524 @@ export interface DFUInfoResponse {
-    ParametersForPaging: string;
-    Filters: string;
-    CacheHint: long;
-    IsSubsetOfFiles: boolean;
-    Warning: string;
-}
-
-export interface DFURecordTypeInfoRequest {
-    Name?: string;
-    IncludeJsonTypeInfo?: boolean;
-    IncludeBinTypeInfo?: boolean;
-}
-
-export interface DFURecordTypeInfoResponse {
-    jsonInfo: string;
-    binInfo: base64Binary;
-}
-
-export interface DFUSearchRequest {
-    ShowExample?: string;
-}
+        ExpirationDate: string;
+    }
+
+    export interface DFUInfoResponse {
+        Exceptions: Exceptions;
+        FileDetail: {
+            Name: string;
+            Filename: string;
+            Prefix: string;
+            NodeGroup: string;
+            NumParts: int;
+            Description: string;
+            Dir: string;
+            PathMask: string;
+            Filesize: string;
+            FileSizeInt64: long;
+            RecordSize: string;
+            RecordCount: string;
+            RecordSizeInt64: long;
+            RecordCountInt64: long;
+            Wuid: string;
+            Owner: string;
+            JobName: string;
+            Persistent: string;
+            Format: string;
+            MaxRecordSize: string;
+            CsvSeparate: string;
+            CsvQuote: string;
+            CsvTerminate: string;
+            CsvEscape: string;
+            Modified: string;
+            Ecl: string;
+            Stat: {
+                MinSkew: string;
+                MaxSkew: string;
+                MinSkewInt64: long;
+                MaxSkewInt64: long;
+                MinSkewPart: long;
+                MaxSkewPart: long;
+            };
+            DFUFilePartsOnClusters: {
+                DFUFilePartsOnCluster: DFUFilePartsOnCluster[];
+            };
+            isSuperfile: boolean;
+            ShowFileContent: boolean;
+            subfiles: {
+                Item: string[];
+            };
+            Superfiles: {
+                DFULogicalFile: DFULogicalFile[];
+            };
+            ProtectList: {
+                DFUFileProtect: DFUFileProtect[];
+            };
+            FromRoxieCluster: boolean;
+            Graphs: {
+                ECLGraph: string[];
+            };
+            UserPermission: string;
+            ContentType: string;
+            CompressedFileSize: long;
+            PercentCompressed: string;
+            IsCompressed: boolean;
+            IsRestricted: boolean;
+            BrowseData: boolean;
+            jsonInfo: string;
+            binInfo: base64Binary;
+            PackageID: string;
+            Partition: {
+                FieldMask: long;
+                FieldNames: {
+                    Item: string[];
+                };
+            };
+            Blooms: {
+                DFUFileBloom: DFUFileBloom[];
+            };
+            ExpireDays: int;
+            KeyType: string;
+            AtRestCost: double;
+            AccessCost: double;
+            ExpirationDate: string;
```

Comment:
Can you open a ticket to display this in ECL Watch?

Reply:
There already was one for this (HPCC-22062)